### ISSUE TITLE
Copy branch id button

### ISSUE THREAD

**Issue by Ciberusps** [playcanvas/editor#879]
Will be awesome to have "copy branch id" button 
With [playcanvas-sync](https://github.com/playcanvas/playcanvas-sync) one of most common actions in editor is copying branch id for `.pcconfig.json` setup

![image](https://user-images.githubusercontent.com/14001879/188493492-15b75efc-7a46-480d-b161-a2d1c5569e47.png)

**Comment by yaustar** [comment]
Wondering if it would be better to have a single copy for all the info (project id and branch id) or separate ones 🤔

**Comment by yaustar** [comment]
Makes sense. For now, I don't know if you knew, you can select the branch id string to copy with Cmd/Ctrl + C
<img width="1098" alt="image" src="https://user-images.githubusercontent.com/16639049/189079445-6d10fb29-ee45-4c83-b5d2-57d8d32cc64b.png">

**Comment by Ciberusps** [comment]
> Makes sense. For now, I don't know if you knew, you can select the branch id string to copy with Cmd/Ctrl + C <img alt="image" width="1098" src="https://user-images.githubusercontent.com/16639049/189079445-6d10fb29-ee45-4c83-b5d2-57d8d32cc64b.png">

yep I know and using it

**Comment by willeastcott** [comment]
Closing as completed — this shipped: the branch switcher context menu has a 'Copy Branch ID' option (src/editor/pickers/version-control/branch-switcher.ts).
